src/assets/styles/config.scss

```diff
@@ -1,29 +1,33 @@
-$theme-blue: #256bff;
-$theme-green: #22C471;
-$theme-red: #E6123F;
-$theme-yellow: #FFB63F;
-$theme-orange: #FF583E;
-$theme-purple: #7F27DE;
-$theme-pink: #FF3E7C;
-$theme-light-gray: #7F7F7F;
+//$theme-blue: #256bff;
+//$theme-green: #22C471;
+//$theme-red: #E6123F;
+//$theme-yellow: #FFB63F;
+//$theme-orange: #FF583E;
+//$theme-purple: #7F27DE;
+//$theme-pink: #FF3E7C;
+
+$theme-red: #9D3D3D;
```

Comment:
Changed a lot of the colors to be more toned down instead of as bright as they were. I also adjusted the default button to be less "bright", and now it's gray.